src/pages/home/index.jsx

```diff
@@ -83,7 +83,7 @@ function HomePage() {
 
 export default HomePage;
 
-const FeatureCard = (props) => {
+const FeatureCard = ({banner,title,content}) => {
```

Comment:
Not required

Reply:
This will be fixed after TS migration of this repo

Reply:
ok

Reply:
Undo these changes as requested

Reply:
ok

Reply:
yeah @pranshuchittora reverted them. thanks